From 0611d8e4d251b79ff3e2402f37cdad7926bc66e9 Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Wed, 18 Apr 2018 11:46:00 -0500
Subject: [PATCH] Roslyn plat assembly paths for RemainingTargets

The crossgen of Microsoft.DotNet.ProjectJsonMigration.dll (a RemainingTargets item) requires Microsoft.CodeAnalysis.CSharp.dll, which is in the RoslynFolders item "Roslyn/bincore/".
---
 src/redist/redist.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/redist/redist.csproj b/src/redist/redist.csproj
index 04b9a7496..8a94089be 100644
--- a/src/redist/redist.csproj
+++ b/src/redist/redist.csproj
@@ -347,7 +347,7 @@
         ReadyToRun="True"
         CreateSymbols="$(CreateCrossgenSymbols)"
         DiasymReaderPath="@(DiasymReaderPath)"
-        PlatformAssemblyPaths="@(PlatformAssemblies);@(RemainingFolders);$(SharedFrameworkNameVersionPath)" />
+        PlatformAssemblyPaths="@(PlatformAssemblies);@(RemainingFolders);@(RoslynFolders);$(SharedFrameworkNameVersionPath)" />
   </Target>
 
   <Target Name="ChmodPublishDir"
-- 
2.16.1.windows.4

